libraries/rtos/rtx/TARGET_ARM7/RTX_Conf_CM.c

```diff
@@ -204,17 +204,12 @@
 /*----------------------------------------------------------------------------
  *      OS Idle daemon
  *---------------------------------------------------------------------------*/
+extern void rtos_idle_loop(void);
```

Comment:
Why did you not just #include "rtos_idle.h" instead to get the function prototype for rtos_idle_loop() since that header contains the prototype as well? Or maybe consider removing the prototype for rtos_idle_loop() from rtos_idle.h so it isn't exposed from the more public header and then keep these forward declarations in the RTX_Conf* sources.